frontend/src/scenes/notebooks/Notebook/notebookLogic.ts

```diff
@@ -534,7 +537,7 @@ export const notebookLogic = kea<notebookLogicType>([
             )
         },
         setLocalContent: async ({ updateEditor, jsonContent }, breakpoint) => {
-            if (values.mode !== 'canvas' && values.notebook?.user_access_level !== 'editor') {
+            if (values.mode !== 'canvas' && !['editor', 'manager'].includes(values.notebook?.user_access_level || '')) {
```

Comment:
Same here, I think using the `accessLevelSatisfied` will make the code easier to read.

(P.S: I'm a fellow contributor to posthog, so please take my code review with a pinch of salt :), its highly possible that I'm not making sense)